Makefile

```diff
@@ -21,6 +21,7 @@ all:
 
 pull:
 	git pull
+	git submodule init
```

Comment:
Oops.  I got spoiled by GitHub Actions `submodules: true`.

Reply:
oooh, I just test it on my local machine, sorry :)

Reply:
On second thought, the `git submodule init` was appropriate with the `pull` target... but not with the replacement `update` target.  That's an external problem to making.

Reply:
commit dropped ;)